src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard";
+import { getArticles } from "./utils/api";
+
+
+const ArticlesList = () => {
+  const [articles, setArticles] = useState([])
+  
```

Comment:
a Loading state would be a very good idea here - this will help the users know what's happening if they have slow internet or the server is taking its time!

Reply:
yessir 🫡